src/core/packages/chrome/browser-internal/src/types.ts

```diff
@@ -151,16 +151,6 @@ export interface InternalChromeStart extends ChromeStart {
      */
     getActiveNavigationNodes$(): Observable<ChromeProjectNavigationNode[][]>;
 
-    /**
-     * Set custom project sidenav component to be used instead of the default project sidenav.
-     * @param component A getter function returning a CustomNavigationComponent.
-     *
-     * @remarks This component will receive Chrome navigation state as props (not yet implemented)
-     *
-     * Use {@link ServerlessPluginStart.setSideNavComponent} to set custom project navigation.
-     */
-    setSideNavComponent(component: SideNavComponent | null): void;
```

Comment:
Previously, we set up our own component from the outside. We duplicated the same code for serverless and stateless setups, but we don't really need to do that. Now, we can just use the component directly from the Chrome service.